nan_json.h

```diff
@@ -78,14 +78,14 @@ class JSON {
     return scope.Escape(parse(json_string));
 #else
 #if NODE_MODULE_VERSION > NODE_6_0_MODULE_VERSION
```

Comment:
We call `Nan::GetCurrentContext()` when `NODE_MODULE_VERSION > NODE_6_0_MODULE_VERSION` today in the `master` branch and build `2.6.1` and nobody is complaining.  That's my reason for thinking it's safe for non-compliant node derivatives.